pkgs/common-updater/scripts/update-source-version

```diff
@@ -188,7 +188,11 @@ tempHash=$(printf '%0*d' "$hashLength" 0)
 if [[ -n "$sri" ]]; then
     # SRI hashes only support base64
     # SRI hashes need to declare the hash type as part of the hash
-    tempHash="$(nix to-sri --type "$oldHashAlgo" "$tempHash")"
+    if [ "$(nix hash --help)" ]; then
```

Comment:
It is probably more reliable to check exit status rather than output.
You can do something like:

```bash
if nix hash to-sri sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA= >/dev/null 2>&1; then
```

Should be probably faster than running man as nix does/did.

Reply:
When we run it anyway we could also convert the hash on the first run, fallback if that fails and error if the fallback fails, too.

Reply:
Yeah, that sounds even more reasonable.